lectures/eigen_II.md

```diff
@@ -397,12 +398,12 @@ This can be proven using what we have learned here.
 With Markov model $M$ with state space $S$ and transition matrix $P$, we can write $P^t$ as
 
 $$
-P^t=\sum_{i=1}^{n-1} \lambda_i^t v_i w_i^{\top}+\mathbb{1} \psi^*,
+P^t=\sum_{i=1}^{n-1} \lambda_i^t v_i w_i^{\top}+1 \cdot \psi^*,
```

Comment:
Is this a good change?  Do we use 1 or \mathbb{1} for a vector of ones?  

Also, is this correct?  The last term should be a square matrix.

Reply:
I think we are trying not to use bold in vectors.(Once discussed when converting Jax version of Newton's method.)

Otherwise we usually state $\mathbb{1}$ represents vector of ones before using them. 

The main reason I am removing this is that it is not stated here.

However this equation comes directly from page 172 equation 4.24 of the networks book.

If you think we need to keep it same with the book we can state it first and then using $\mathbb{1}$?

Reply:
Thanks @HengchengZhang , I undstand.

Let's agree 1 is a special case and use $\mathbb 1$ for a vector or matrix of ones ---  and state this the first time it occurs in each lecture.  Please adjust and note this in the manual.

Please also see my second comment about the square matrix before you resolve this conversation.

Reply:
Yes I will update the manual and also the Jax version of Newton's method as well.

In the networks book, $\psi^*$ and $\mathbb{1}$ here are left and right eigenvector associate with $P$.

Since it is strongly connected and aperiodic, $\lim_{t \rightarrow} P^* = $

Reply:
I see, so $\mathbb 1$ is $n \times 1$ and $\psi^*$ is $1 \times n$, so the product is $n \times n$.

This is fine, as long as the shapes are clear in the lecture.